app/pages/project/metadata.jsx

```diff
@@ -44,25 +44,59 @@ class ProjectMetadata extends React.Component {
     }
   }
 
+  renderStatus() {
+    let percentComplete = 0;
+    this.props.activeWorkflows.map((workflow) => {
+      percentComplete += workflow.completeness;
+      return percentComplete;
+    });
+
+    if (this.props.activeWorkflows.length > 0) {
+      percentComplete /= this.props.activeWorkflows.length;
+    }
+
+    return (
+      <div className="project-metadata-status-bar">
+        <svg width="100%" height="1em" viewBox="0 0 1 1" preserveAspectRatio="none" style={{ display: 'block' }}>
+          <defs>
+            <linearGradient id="linear-gradient">
+              <stop offset="14%" stopColor="#E45950" stopOpacity="0%" />
+              <stop offset="79%" stopColor="#F0B200" stopOpacity="85%" />
+            </linearGradient>
+          </defs>
+          <rect fill="url(#linear-gradient)" stroke="none" x="0" y="0" width={percentComplete} height="1" />
+          <rect fill="hsl(0, 0%, 75%)" stroke="none" x={percentComplete} y="0" width={1 - percentComplete} height="1" />
+        </svg>
+        <br />
+        {Math.floor(percentComplete * 100)}% Complete
+      </div>
+    );
+  }
+
   render() {
     const project = this.props.project;
     const statsLink = `/projects/${project.slug}/stats`;
 
     return (
-      <div className="project-metadata content-container">
-        <div className="project-metadata-header">
-          <span>{project.display_name}</span>{' '}
+      <div className="project-home-page__container">
+        <div className="project-metadata">
           <Link to={statsLink}>
-            <span>Statistics</span>
+            <span>{project.display_name}{' '}Statistics</span>
```

Comment:
This should be styled so that it's obviously a link. At present, it looks like the other headings, so it isn't clear that you can click it.